test/api/interop_extra_test.js

```diff
@@ -51,7 +51,7 @@ function echoMetadataGenerator(options, callback) {
 
 const credentials = grpc.credentials.createFromMetadataGenerator(echoMetadataGenerator);
 
-describe(`${anyGrpc.clientName} client -> ${anyGrpc.serverName} server`, function() {
+describe.only(`${anyGrpc.clientName} client -> ${anyGrpc.serverName} server`, function() {
```

Comment:
Is this `only` intentional?

Reply:
No it is not. I left that in there from when I was fixing the test. Thanks for catching it.